### PR TITLE
fix: Fix name description

### PR DESCRIPTION
**Pull request by leoendless**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubesphere/kubesphere/issues/2340

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by leoendless** [comment]
I found that in K8s it is possible to create a resource starting with a number. We should not limit this feature.

File: src/locales/en/application.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default {
   LB_RANDOM: 'RANDOM',
 
   CLUSTER_NAME_DESC:
-    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin with a lowercase letter, ending with a number or lowercase letter. The maximum length of characters is set to 14.',
+    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin and ending with a lowercase letter or a number. The maximum length of characters is set to 14.',
```
Comment (Sherlock113) [inline src/locales/en/application.js:34]: and must begin and **ending** with a lowercase letter or a number -> and must begin and **end** with a lowercase letter or a number

File: src/locales/en/base.js [hunk 2]
```diff
@@ -22,9 +22,9 @@ export default {
   DETAILS: 'Details',
   NODES: 'Nodes',
   NAME_DESC:
-    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin with a lowercase letter, ending with a number or lowercase letter. The maximum length of characters is set to 63.',
+    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin and ending with a lowercase letter or a number. The maximum length of characters is set to 63.',
```
Comment (Sherlock113) [inline src/locales/en/base.js:25]: ending -> end

File: src/locales/en/base.js [hunk 3]
```diff
@@ -22,9 +22,9 @@ export default {
...
   LONG_NAME_DESC:
-    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin with a lowercase letter, ending with a number or lowercase letter. The maximum length of characters is set to 253.',
+    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin and ending with a lowercase letter or a number. The maximum length of characters is set to 253.',
```
Comment (Sherlock113) [inline src/locales/en/base.js:27]: ending -> end

File: src/locales/en/service.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ export default {
   SERVICE_SELECTOR_AFFECT_2: ' affect {count} workloads',
 
   SERVICE_NAME_DESC:
-    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin with a lowercase letter, ending with a number or lowercase letter. The maximum length of characters is set to 63.',
+    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin and ending with a lowercase letter or a number. The maximum length of characters is set to 63.',
```
Comment (Sherlock113) [inline src/locales/en/service.js:28]: ending -> end

File: src/locales/en/workload.js [hunk 5]
```diff
@@ -40,7 +40,7 @@ export default {
   CRONJOB_CREATE_DESC:
     'A CronJob creates Jobs on a time-based schedule. For example, it runs a job periodically on a given schedule or only runs a job once on a given time.',
   CRONJOB_NAME_DESC:
-    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin with a lowercase letter, ending with a number or lowercase letter. The maximum length of characters is set to 52.',
+    'It can only contain lowercase letters, numbers and hyphens("-"), and must begin and ending with a lowercase letter or a number. The maximum length of characters is set to 52.',
```
Comment (Sherlock113) [inline src/locales/en/workload.js:43]: ending -> end

File: src/locales/en/workload.js [hunk 6]
```diff
@@ -99,9 +99,9 @@ export default {
   PORT_INPUT_DESC:
     'The name is required and cannot be duplicated when there are multiple ports,',
   PORT_NAME_DESC:
-    'The port name can only contain lowercase letters, numbers and hyphens("-"), and must begin with a lowercase letter, ending with a number or lowercase letter. The maximum length of characters is set to 63.',
+    'The port name can only contain lowercase letters, numbers and hyphens("-"), and must begin and ending with a lowercase letter or a number. The maximum length of characters is set to 63.',
```
Comment (Sherlock113) [inline src/locales/en/workload.js:102]: ending -> end

File: src/locales/en/workload.js [hunk 7]
```diff
@@ -99,9 +99,9 @@ export default {
...
   WORKLOAD_PORT_NAME_DESC:
-    'The port name can only contain lowercase letters, numbers and hyphens("-"), and must begin with a lowercase letter, ending with a number or lowercase letter. The maximum length of characters is set to 15.',
+    'The port name can only contain lowercase letters, numbers and hyphens("-"), and must begin and ending with a lowercase letter or a number. The maximum length of characters is set to 15.',
```
Comment (Sherlock113) [inline src/locales/en/workload.js:104]: ending -> end

**Pull request merged**